src/containers/ViewerPanel/index.tsx

```diff
@@ -222,7 +224,7 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
             setBuffering,
             setIsPlaying,
         } = this.props;
-        let newTime = time;
+        let newTime = timeOverride || time;
```

Comment:
what if timeOverride is zero vs undefined?

Reply:
for optional numbers you might have to do an explicit undefined check instead of this falsy-ness check

Reply:
That is a really good point, thanks!